src/lib/__tests__/index.ts

```diff
@@ -13,98 +13,114 @@ const stubDataPath = './data';
+//   installContracts();
+
+//   expect(execSh).toHaveBeenCalledTimes(1);
+//   expect(execSh.mock.calls[0][0])
+//     .toBe(`java -jar ${path.resolve(specmaticJarPathLocal)} install`);
+// });
+
+// test('installSpecs installs contracts to local', () => {
+//   installSpecs();
+
+//   expect(execSh).toHaveBeenCalledTimes(1);
+//   expect(execSh.mock.calls[0][0])
+//     .toBe(`java -jar ${path.resolve(specmaticJarPathLocal)} install`);
+// });
+
+// test('loadDynamicStub with default baseUrl', () => {
+//   fetch.mockReturnValue(Promise.resolve("{}"));
+//   loadDynamicStub(path.resolve('./mockStub.json'));
+
+//   expect(fetch).toHaveBeenCalledTimes(1);
+//   expect(fetch.mock.calls[0][0]).toBe('http://localhost:9000/_specmatic/expectations');
+//   expect(fetch.mock.calls[0][1]).toMatchObject({
+//     method: 'POST',
+//     body: JSON.stringify(mockStub)
+//   });
+// });
+
+// test('setExpectations with default baseUrl', () => {
+//   fetch.mockReturnValue(Promise.resolve("{}"));
+//   setExpectations(path.resolve('./mockStub.json'));
+
+//   expect(fetch).toHaveBeenCalledTimes(1);
+//   expect(fetch.mock.calls[0][0]).toBe('http://localhost:9000/_specmatic/expectations');
+//   expect(fetch.mock.calls[0][1]).toMatchObject({
+//     method: 'POST',
+//     body: JSON.stringify(mockStub)
+//   });
+// });
+
+// test('loadDynamicStub with a different baseUrl for the stub server', () => {
+//   fetch.mockReturnValue(Promise.resolve("{}"));
+//   const stubServerBaseUrl = 'http://localhost:8000/'
+//   loadDynamicStub(path.resolve('./mockStub.json'), stubServerBaseUrl);
+
+//   expect(fetch).toHaveBeenCalledTimes(1);
+//   expect(fetch.mock.calls[0][0]).toBe(`${stubServerBaseUrl}_specmatic/expectations`);
+//   expect(fetch.mock.calls[0][1]).toMatchObject({
+//     method: 'POST',
+//     body: JSON.stringify(mockStub)
+//   });
+// });
+
+// test('setExpectations with a different baseUrl for the stub server', () => {
+//   fetch.mockReturnValue(Promise.resolve("{}"));
+//   const stubServerBaseUrl = 'http://localhost:8000/'
+//   setExpectations(path.resolve('./mockStub.json'), stubServerBaseUrl);
+
+//   expect(fetch).toHaveBeenCalledTimes(1);
+//   expect(fetch.mock.calls[0][0]).toBe(`${stubServerBaseUrl}_specmatic/expectations`);
+//   expect(fetch.mock.calls[0][1]).toMatchObject({
+//     method: 'POST',
+//     body: JSON.stringify(mockStub)
+//   });
+// });
+
+test('setSpecmaticEnvVariable updates the environment variable value in the specmatic.json file', () => {
+  const variableObj1: Record<string, string> = { "key1": "updated-value1", "key2": "updated-value2", "key3": "updated-value3", "key4": "updated-value4" }
+  const variableObj2: Record<string, string> = { "key1": "initial-value1", "key2": "initial-value2", "key3": "initial-value3", "key4": "initial-value4" }
+
+  expect(setSpecmaticEnvironement("variables", variableObj1)).toBe(true)
+  expect(setSpecmaticEnvironement("variables", variableObj2)).toBe(true)
```

Comment:
we still need to add assertions after reading the file, if the correct things were actually set